src/Components/App/App.tsx

```diff
@@ -15,6 +15,7 @@ const App: FC = () => {
   const [warning, setWarning] = useState("");
   const [currentCity, setCurrentCity] = useState<WeatherLocation | null>(null);
   const [forecastDetails, setForecastDetails] = useState<Forecast | null>(null);
+  const [stateString, setStateString] = useState<State | null>(null);;
```

Comment:
Nice! I wonder if naming would be clearer without using data type of string in name?